# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: ipc/ipdl/sync-messages.ini
# Commit: 57607f8b6d8e
# Full Hash: 57607f8b6d8e1619b5c488494fd57547944c47b1
# Author: Dan Glastonbury <dan.glastonbury@gmail.com>
# Date: 2020-10-21 09:51:28
# Regressor Bug: 1595994
# File Overlap Count: 1
# Description:
#   Bug 1595994 - P7: Add Supports messages to PRemoteDecoderManager. r=kamidphish,nika,ipc-reviewers
#   
#   Add a synchronous Supports message to the IPDL PRemoteDecoderManager protocol so
#   decoder modules can query for playback support in the actual process that will
#   attempt to do the decoding.
# ==============================================================================

diff -r 1932f2f34993 -r 57607f8b6d8e ipc/ipdl/sync-messages.ini
--- a/ipc/ipdl/sync-messages.ini	Tue Oct 20 23:26:25 2020 +0000
+++ b/ipc/ipdl/sync-messages.ini	Tue Oct 20 23:26:27 2020 +0000
@@ -884,6 +884,8 @@
 description = legacy sync IPC - please add detailed description
 [PRemoteDecoderManager::PRemoteDecoder]
 description = See Bug 1505976 - investigate changing to async instead of matching GPU pattern
+[PRemoteDecoderManager::Supports]
+description = See Bug 1672072 - We will always need a synchronous call in the worse case scenario, follow-up calls will be cached; however it could be made fully asynchronous under some circumstances..
 [PContent::LaunchRDDProcess]
 description = See Bug 1518344 - investigate using async for PContent::LaunchRDDProcess
 [PRemoteDecoderManager::Readback]
